packages/mongodb-chatbot-ui/src/useConversation.tsx

```diff
@@ -118,11 +128,15 @@ function conversationReducer(
       if (!state.conversationId) {
         console.error(`Cannot addMessage without a conversationId`);
       }
-      const newMessage = createMessage(
-        action.role,
-        action.content,
-        action.metadata
-      );
+
+      const { type: _, role, content, ...assistantMessageData } = action;
```

Comment:
can you remove `type: _`?

Reply:
I actually think I can just delete the whole line and use `action` directly